### PR TITLE
fix parameter files

### PR DESCRIPTION
**Pull request by dgergel**
Quick fix to ACCESS-ESM1-5 tasmin and pr files.

### CODE REVIEW

**Comment by brews** [comment]
Afraid this doesn't work either, @dgergel. I took a look and there appears to be no tasmin or tasmax for ssp370 or ssp270:

```python
cmip6.query("source_id == 'ACCESS-ESM1-5' and variable_id in ['tasmax', 'tasmax'] and experiment_id in ['ssp270', 'ssp370']")
#Empty DataFrame
#Columns: [activity_id, institution_id, source_id, experiment_id, member_id, table_id, variable_id, grid_label, zstore, #dcpp_init_year, version]
#Index: []
```

They do have some records for ssp585, though. For what that's worth.

**Comment by dgergel** [comment]
@brews there isn't an ssp270 - do you mean ssp245? And there definitely is tasmax for ssp245 and 370 - I checked the archive again for this and they are def there. There may be some other issue with the parameter file however

**Comment by dgergel** [comment]
@brews for additional cmip6 archive ref see here: https://gist.github.com/dgergel/52d942cc4f191f542189c28e125ad9c4

File: workflows/parameters/ACCESS-ESM1-5-tasmin.yaml [hunk 1]
```diff
@@ -4,13 +4,13 @@ jobs: |
       "target": "historical",
       "variable_id": "tasmin",
       "historical": { "activity_id": "CMIP", "experiment_id": "historical", "table_id": "day", "variable_id": "tasmin", "source_id": "ACCESS-ESM1-5", "institution_id": "CSIRO", "member_id": "r1i1p1f1", "grid_label": "gn", "version": "20191115" },
-      "ssp": { "activity_id": "ScenarioMIP", "experiment_id": "ssp245", "table_id": "day", "variable_id": "tasmin", "source_id": "ACCESS-ESM1-5", "institution_id": "CSIRO", "member_id": "r1i1p1f1", "grid_label": "gn", "version": "20191115" }
+      "ssp": { "activity_id": "ScenarioMIP", "experiment_id": "ssp370", "table_id": "day", "variable_id": "tasmin", "source_id": "ACCESS-ESM1-5", "institution_id": "CSIRO", "member_id": "r1i1p1f1", "grid_label": "gn", "version": "20191115" }
```
Comment (brews) [inline workflows/parameters/ACCESS-ESM1-5-tasmin.yaml:7]:
```suggestion
      "ssp": { "activity_id": "ScenarioMIP", "experiment_id": "ssp370", "table_id": "day", "variable_id": "tasmin", "source_id": "ACCESS-ESM1-5", "institution_id": "CSIRO", "member_id": "r1i1p1f1", "grid_label": "gn", "version": "20191108" }
```

File: workflows/parameters/ACCESS-ESM1-5-tasmin.yaml [hunk 2]
```diff
@@ -4,13 +4,13 @@ jobs: |
...
     },
     {
       "target": "ssp",
       "variable_id": "tasmin",
-      "historical": { "activity_id": "CMIP", "experiment_id": "historical", "table_id": "day", "variable_id": "tasmax", "source_id": "ACCESS-ESM1-5", "institution_id": "CSIRO", "member_id": "r1i1p1f1", "grid_label": "gn", "version": "20191115" },
-      "ssp": { "activity_id": "ScenarioMIP", "experiment_id": "ssp245", "table_id": "day", "variable_id": "tasmin", "source_id": "ACCESS-ESM1-5", "institution_id": "CSIRO", "member_id": "r1i1p1f1", "grid_label": "gn", "version": "20191115" }
+      "historical": { "activity_id": "CMIP", "experiment_id": "historical", "table_id": "day", "variable_id": "tasmin", "source_id": "ACCESS-ESM1-5", "institution_id": "CSIRO", "member_id": "r1i1p1f1", "grid_label": "gn", "version": "20191115" },
+      "ssp": { "activity_id": "ScenarioMIP", "experiment_id": "ssp370", "table_id": "day", "variable_id": "tasmin", "source_id": "ACCESS-ESM1-5", "institution_id": "CSIRO", "member_id": "r1i1p1f1", "grid_label": "gn", "version": "20191115" }
```
Comment (brews) [inline workflows/parameters/ACCESS-ESM1-5-tasmin.yaml:13]:
```suggestion
      "ssp": { "activity_id": "ScenarioMIP", "experiment_id": "ssp370", "table_id": "day", "variable_id": "tasmin", "source_id": "ACCESS-ESM1-5", "institution_id": "CSIRO", "member_id": "r1i1p1f1", "grid_label": "gn", "version": "20191108" }
```

File: workflows/parameters/ACCESS-ESM1-5-tasmin.yaml [hunk 3]
```diff
@@ -4,13 +4,13 @@ jobs: |
...
     },
     {
       "target": "ssp",
       "variable_id": "tasmin",
-      "historical": { "activity_id": "CMIP", "experiment_id": "historical", "table_id": "day", "variable_id": "tasmax", "source_id": "ACCESS-ESM1-5", "institution_id": "CSIRO", "member_id": "r1i1p1f1", "grid_label": "gn", "version": "20191115" },
-      "ssp": { "activity_id": "ScenarioMIP", "experiment_id": "ssp245", "table_id": "day", "variable_id": "tasmin", "source_id": "ACCESS-ESM1-5", "institution_id": "CSIRO", "member_id": "r1i1p1f1", "grid_label": "gn", "version": "20191115" }
+      "historical": { "activity_id": "CMIP", "experiment_id": "historical", "table_id": "day", "variable_id": "tasmin", "source_id": "ACCESS-ESM1-5", "institution_id": "CSIRO", "member_id": "r1i1p1f1", "grid_label": "gn", "version": "20191115" },
```
Comment (brews) [inline workflows/parameters/ACCESS-ESM1-5-tasmin.yaml:12]:
```suggestion
      "historical": { "activity_id": "CMIP", "experiment_id": "historical", "table_id": "day", "variable_id": "tasmin", "source_id": "ACCESS-ESM1-5", "institution_id": "CSIRO", "member_id": "r1i1p1f1", "grid_label": "gn", "version": "20191108" },
```

File: workflows/parameters/ACCESS-ESM1-5-tasmin.yaml [hunk 4]
```diff
@@ -23,5 +23,11 @@ jobs: |
       "variable_id": "tasmin",
       "historical": { "activity_id": "CMIP", "experiment_id": "historical", "table_id": "day", "variable_id": "tasmin", "source_id": "ACCESS-ESM1-5", "institution_id": "CSIRO", "member_id": "r1i1p1f1", "grid_label": "gn", "version": "20191115" },
       "ssp": { "activity_id": "ScenarioMIP", "experiment_id": "ssp126", "table_id": "day", "variable_id": "tasmin", "source_id": "ACCESS-ESM1-5", "institution_id": "CSIRO", "member_id": "r1i1p1f1", "grid_label": "gn", "version": "20191115" }
+    },
+    {
+      "target": "ssp",
+      "variable_id": "tasmin",
+      "historical": { "activity_id": "CMIP", "experiment_id": "historical", "table_id": "day", "variable_id": "tasmin", "source_id": "ACCESS-ESM1-5", "institution_id": "CSIRO", "member_id": "r1i1p1f1", "grid_label": "gn", "version": "20191115" },
+      "ssp": { "activity_id": "ScenarioMIP", "experiment_id": "ssp585", "table_id": "day", "variable_id": "tasmin", "source_id": "ACCESS-ESM1-5", "institution_id": "CSIRO", "member_id": "r1i1p1f1", "grid_label": "gn", "version": "20191115" }
```
Comment (brews) [inline workflows/parameters/ACCESS-ESM1-5-tasmin.yaml:31]:
```suggestion
      "ssp": { "activity_id": "ScenarioMIP", "experiment_id": "ssp585", "table_id": "day", "variable_id": "tasmin", "source_id": "ACCESS-ESM1-5", "institution_id": "CSIRO", "member_id": "r1i1p1f1", "grid_label": "gn", "version": "20210317" }
```

File: workflows/parameters/ACCESS-ESM1-5-tasmin.yaml [hunk 5]
```diff
@@ -23,5 +23,11 @@ jobs: |
       "variable_id": "tasmin",
       "historical": { "activity_id": "CMIP", "experiment_id": "historical", "table_id": "day", "variable_id": "tasmin", "source_id": "ACCESS-ESM1-5", "institution_id": "CSIRO", "member_id": "r1i1p1f1", "grid_label": "gn", "version": "20191115" },
       "ssp": { "activity_id": "ScenarioMIP", "experiment_id": "ssp126", "table_id": "day", "variable_id": "tasmin", "source_id": "ACCESS-ESM1-5", "institution_id": "CSIRO", "member_id": "r1i1p1f1", "grid_label": "gn", "version": "20191115" }
+    },
+    {
+      "target": "ssp",
+      "variable_id": "tasmin",
+      "historical": { "activity_id": "CMIP", "experiment_id": "historical", "table_id": "day", "variable_id": "tasmin", "source_id": "ACCESS-ESM1-5", "institution_id": "CSIRO", "member_id": "r1i1p1f1", "grid_label": "gn", "version": "20191115" },
```
Comment (brews) [inline workflows/parameters/ACCESS-ESM1-5-tasmin.yaml:30]:
```suggestion
      "historical": { "activity_id": "CMIP", "experiment_id": "historical", "table_id": "day", "variable_id": "tasmin", "source_id": "ACCESS-ESM1-5", "institution_id": "CSIRO", "member_id": "r1i1p1f1", "grid_label": "gn", "version": "20191108" },
```

File: workflows/parameters/ACCESS-ESM1-5-pr.yaml [hunk 6]
```diff
@@ -23,5 +23,11 @@ jobs: |
       "variable_id": "pr",
       "historical": { "activity_id": "CMIP", "experiment_id": "historical", "table_id": "day", "variable_id": "pr", "source_id": "ACCESS-ESM1-5", "institution_id": "CSIRO", "member_id": "r1i1p1f1", "grid_label": "gn", "version": "20191115" },
       "ssp": { "activity_id": "ScenarioMIP", "experiment_id": "ssp126", "table_id": "day", "variable_id": "pr", "source_id": "ACCESS-ESM1-5", "institution_id": "CSIRO", "member_id": "r1i1p1f1", "grid_label": "gn", "version": "20191115" }
+    },
+    {
+      "target": "ssp",
+      "variable_id": "pr",
+      "historical": { "activity_id": "CMIP", "experiment_id": "historical", "table_id": "day", "variable_id": "pr", "source_id": "ACCESS-ESM1-5", "institution_id": "CSIRO", "member_id": "r1i1p1f1", "grid_label": "gn", "version": "20191115" },
```
Comment (brews) [inline workflows/parameters/ACCESS-ESM1-5-pr.yaml:30]:
```suggestion
      "historical": { "activity_id": "CMIP", "experiment_id": "historical", "table_id": "day", "variable_id": "pr", "source_id": "ACCESS-ESM1-5", "institution_id": "CSIRO", "member_id": "r1i1p1f1", "grid_label": "gn", "version": "20191108" },
```

File: workflows/parameters/ACCESS-ESM1-5-pr.yaml [hunk 7]
```diff
@@ -23,5 +23,11 @@ jobs: |
...
+      "ssp": { "activity_id": "ScenarioMIP", "experiment_id": "ssp585", "table_id": "day", "variable_id": "pr", "source_id": "ACCESS-ESM1-5", "institution_id": "CSIRO", "member_id": "r1i1p1f1", "grid_label": "gn", "version": "20191115" }
```
Comment (brews) [inline workflows/parameters/ACCESS-ESM1-5-pr.yaml:31]:
```suggestion
      "ssp": { "activity_id": "ScenarioMIP", "experiment_id": "ssp585", "table_id": "day", "variable_id": "pr", "source_id": "ACCESS-ESM1-5", "institution_id": "CSIRO", "member_id": "r1i1p1f1", "grid_label": "gn", "version": "20210317" }
```

**Review by brews — APPROVED** [review]
~@dgergel Thanks for the clarification. This PR is helpful. 👍  The version_id needs to be updated for this. Aside from that I think this can merge.~

File: workflows/parameters/ACCESS-ESM1-5-pr.yaml (hunk 7 above)
Comment (brews) [reply workflows/parameters/ACCESS-ESM1-5-pr.yaml:31]: Wait. This is wrong. Cancel this. Sorry.

File: workflows/parameters/ACCESS-ESM1-5-pr.yaml (hunk 6 above)
Comment (brews) [reply workflows/parameters/ACCESS-ESM1-5-pr.yaml:30]: Wait. This is wrong. Cancel this. Sorry.

File: workflows/parameters/ACCESS-ESM1-5-tasmin.yaml (hunk 5 above)
Comment (brews) [reply workflows/parameters/ACCESS-ESM1-5-tasmin.yaml:30]: Wait. This is wrong. Cancel this. Sorry.

File: workflows/parameters/ACCESS-ESM1-5-tasmin.yaml (hunk 4 above)
Comment (brews) [reply workflows/parameters/ACCESS-ESM1-5-tasmin.yaml:31]: Wait. This is wrong. Cancel this. Sorry.

File: workflows/parameters/ACCESS-ESM1-5-tasmin.yaml (hunk 3 above)
Comment (brews) [reply workflows/parameters/ACCESS-ESM1-5-tasmin.yaml:12]: Wait. This is wrong. Cancel this. Sorry.

File: workflows/parameters/ACCESS-ESM1-5-tasmin.yaml (hunk 2 above)
Comment (brews) [reply workflows/parameters/ACCESS-ESM1-5-tasmin.yaml:13]: Wait. This is wrong. Cancel this. Sorry.

File: workflows/parameters/ACCESS-ESM1-5-tasmin.yaml (hunk 1 above)
Comment (brews) [reply workflows/parameters/ACCESS-ESM1-5-tasmin.yaml:7]: Wait. This is wrong. Cancel this. Sorry.

**Comment by brews** [comment]
Scratch that, @dgergel . I was looking at the wrong ACCESS. Very sorry. My mistake.

**Comment by brews** [comment]
Ahh, the problem is that you're looking at the non-QC'd entries. We were only grabbing from QC'd data. I changed this in #337 and this should be resolved.

**Comment by brews** [comment]
Merging this.

**Pull request merged**